### PR TITLE
Issue #204,202,189 - Update Auto Import Logic

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This should: 
**(done)** 1. display the footer content only when canUndo (ref: https://bugzilla.mozilla.org/show_bug.cgi?id=1271799#c14) resolves as true
**(done)** 2. update the logic for calling the undo function after the canUndo promise resolves as true
**(done)** 3. Direct the folks who click "No Thanks" to opt out of onboarding to about:newtab rather than about:home.
**(done)** 4. Remove logic for manual/user import sidebar (#202)
**(done)** 5. Modify text for new-tab footer (#189)

Issue Tracker: #204

### CODE REVIEW

**Comment by FrancescoSTL** [comment]
Note: In order to mitigate the issue of users clicking the actionButton before they've received their first sidebar task, @chrismore and I have decided it would be best to only show the actionButton once the first task is assigned. That will be reflected in this PR

**Comment by chrismore** [comment]
I just reviewed this and it functionally works as planned! Nice job r+ from a functionality perspective.

File: data/js/about-newtab.js [hunk 1]
```diff
@@ -2,19 +2,30 @@
...
 function showUserData(headerContent, footerContent) {
     headDiv.insertAdjacentHTML('beforebegin', headerContent);
     footDiv.insertAdjacentHTML('beforebegin', footerContent);
+
+    /* staging for auto migrate
+    var undoMigrate = document.querySelector('#undo-migrate');
+
+    undoMigrate.addEventListener('click', function() {
+	    addon.port.emit('undoMigrate');
+	});*/
```
Comment (schalkneethling) [inline data/js/about-newtab.js]: Nit: Even though this is commented out at the moment, just double check your indentation.

File: data/js/about-newtab.js [hunk 2]
```diff
@@ -2,19 +2,30 @@
...
-undoMigrate.addEventListener('click', function() {
-    addon.port.emit('undoMigrate');
 });
+
+// listen for the remove footer event
+self.port.on('removeFooter', function() {
+	// we can only grab this after it is added to the page, which it should anytime removeFooter is emitted
+    var footer = document.querySelector('#footer');
+    // hide the footer content
+    footer.style.visibility = "hidden";
```
Comment (schalkneethling) [inline data/js/about-newtab.js]: Nit: Use single quotes

File: data/js/about-newtab.js [hunk 3]
```diff
@@ -2,19 +2,30 @@
...
+});
```
Comment (schalkneethling) [inline data/js/about-newtab.js]: Nit: No empty line at end of file

**Comment by schalkneethling** [comment]
There is still some things to clean up but, I will open a PR for that separately. I would like to have this merged and running locally as part of the clickable tokens work. Thanks @FrancescoSTL r+

**Pull request merged**